pkg/controller/migplan/validation.go

```diff
@@ -94,6 +95,7 @@ const (
 	StorageNotReadyMessage                = "The referenced `migStorageRef` does not have a `Ready` condition."
 	NsListEmptyMessage                    = "The `namespaces` list may not be empty."
 	InvalidDestinationClusterMessage      = "The `srcMigClusterRef` and `dstMigClusterRef` cannot be the same."
+	NsNotSupported                        = "Namespaces [] not supported. See: `unsupportedNamespaces` for details."
```

Comment:
Since there is a separate list (which may be large) in the status, let's not list them on the condition as well.